boards/sifli/sf32lb52_devkit_lcd/sf32lb52_devkit_lcd.yaml

```diff
@@ -14,4 +14,5 @@ supported:
   - gpio
   - watchdog
   - bluetooth
+  - display
```

Comment:
not true, please add a shield

Reply:
Will the build_all trigger for this driver without the display tag on the board supposed to be used for the driver build test? Eg like the bl61x one: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/aithinker/ai_m62_12f_kit/ai_m62_12f_kit.yaml ? Or is the bl61x one is doing it wrong (it has no display on the board either).

Reply:
Aight, will look into it.